projects/sbb-esta/angular-business/dialog/dialog.md

```diff
@@ -0,0 +1,166 @@
+The dialog can be used to seek confirmation as see below
+
+```html
+<div sbbDialog>
+  <div sbbDialogHeader>
+    <p>Hi {{ data.name }}</p>
+  </div>
+  <div sbbDialogContent>
+    <div>
+      What's your favorite animal?
+      <sbb-field label="Animal" mode="long">
+        <input type="text" [(ngModel)]="data.animal" cdkFocusInitial />
+      </sbb-field>
+    </div>
+  </div>
+  <div sbbDialogFooter>
+    <button sbbButton [sbbDialogClose]="data.animal" cdkFocusInitial>
```

Comment:
I did not notice this in the lightbox, but using cdkFocusInitial twice does not seem to make sense?
Could you remove this one?

Reply:
Done!